### PR TITLE
Update links to etherscan.io

### PR DESCRIPTION
**Pull request by dstarcev**
- Update links to etherscan.io to reflect recent changes (e.g. separate domain for ropsten)
- Remove duplicated code
- Add some tests

### CODE REVIEW

File: js/src/3rdparty/etherscan/links.js [hunk 1]
```diff
@@ -15,15 +15,18 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 // NOTE: Keep 'isTestnet' for backwards library compatibility
-export const url = (isTestnet = false, netVersion = '0') => {
-  let prefix = '';
+export const url = (isTestnet = false, netVersion = '0', defaultPrefix = '') => {
+  let prefix = defaultPrefix;
 
   switch (netVersion) {
     case '2':
```
Comment (5chdn) [inline js/src/3rdparty/etherscan/links.js]: we should remove `'2'` completely from this list, I don't know of any working morden explorer and etherscan recently told me they do not plan to maintain one.

**Comment by dstarcev** [comment]
I have removed `morden` and added the recently born `rinkeby`. Parity doesn't support clique yet but when (and if) it does, correct links will already be there.

**Comment by rphmeier** [comment]
Doesn't ETC still use morden as a testnet?

**Comment by gavofyork** [comment]
yeah i think so.

**Comment by dstarcev** [comment]
As @5chdn mentioned, etherscan.io doesn't seem to support morden anymore. https://morden.etherscan.io/ refers to `ROPSTEN (Attacked) TESTNET` and I am not sure if we should provide links to it.

**Comment by 5chdn** [comment]
> Doesn't ETC still use morden as a testnet?

It's not about removing the morden testnet but the explorer links (as there is none).

**Comment by dstarcev** [comment]
I have updated the PR as the mainnet network id is actually `1`.

**Comment by jacogr** [comment]
Looks good, tested as well. Just waiting on CLA signoff.

**Comment by dstarcev** [comment]
[clabot:check]

**Pull request merged**